lib/matplotlib/figure.py

```diff
@@ -388,6 +388,11 @@ def suptitle(self, t, **kwargs):
                 'size': 'figure.titlesize', 'weight': 'figure.titleweight'}
         return self._suplabels(t, info, **kwargs)
 
+    def get_suptitle(self):
+        """Return the suptitle as string or an empty string if not set."""
+        text_obj = self._suptitle
+        return "" if text_obj is None else text_obj.get_text()
```

Comment:
do we want to just return the text object?  Or keep as close to `get_title` as possible?

Reply:
I want so keep close to `get_title` because of consistency, but also because of the symmertry

```
fig.suptitle(somestr)
fig.get_suptitle() == somestr
```

So far, nobody has complained that `get_title` does not return a `Text`. I believe quite the opposite: Most people don't know about the underlying `Text` and consider these things as simple strings. Yes, you still cannot access the text properties, but as said, nobody has complained about that for `get_title`. I would care about that when the need arises.